Sat, 22 Apr 2017 17:03:16 +0100 | INFO   | server     |  | Serving using application   : /usr/bin/python -u some-game.py
Sat, 22 Apr 2017 17:03:16 +0100 | INFO   | server     |  | Starting WebSocket server   : ws://adamk117-System-Product-Name:60001/
Sat, 22 Apr 2017 17:17:08 +0100 | ACCESS | session    | url:'http://localhost:60001/' id:'1492877828778991010' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' | CONNECT
Sat, 22 Apr 2017 17:17:14 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492877828778991010' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29524' | Traceback (most recent call last):
Sat, 22 Apr 2017 17:17:14 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492877828778991010' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29524' |   File "some-game.py", line 7, in <module>
Sat, 22 Apr 2017 17:17:14 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492877828778991010' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29524' |     name = raw_input()
Sat, 22 Apr 2017 17:17:14 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492877828778991010' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29524' | EOFError: EOF when reading a line
Sat, 22 Apr 2017 17:17:14 +0100 | ACCESS | session    | url:'http://localhost:60001/' id:'1492877828778991010' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29524' | DISCONNECT
Sat, 22 Apr 2017 17:22:29 +0100 | ACCESS | session    | url:'http://localhost:60001/' id:'1492878149579660585' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' | CONNECT
Sat, 22 Apr 2017 17:22:35 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492878149579660585' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29611' | Traceback (most recent call last):
Sat, 22 Apr 2017 17:22:35 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492878149579660585' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29611' |   File "some-game.py", line 7, in <module>
Sat, 22 Apr 2017 17:22:35 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492878149579660585' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29611' |     name = raw_input()
Sat, 22 Apr 2017 17:22:35 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492878149579660585' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29611' | EOFError: EOF when reading a line
Sat, 22 Apr 2017 17:22:35 +0100 | ACCESS | session    | url:'http://localhost:60001/' id:'1492878149579660585' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'29611' | DISCONNECT
Sat, 22 Apr 2017 18:30:30 +0100 | ACCESS | session    | url:'http://localhost:60001/' id:'1492882230049400170' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' | CONNECT
Sat, 22 Apr 2017 18:30:31 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492882230049400170' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'30215' | Traceback (most recent call last):
Sat, 22 Apr 2017 18:30:31 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492882230049400170' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'30215' |   File "some-game.py", line 7, in <module>
Sat, 22 Apr 2017 18:30:31 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492882230049400170' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'30215' |     name = raw_input()
Sat, 22 Apr 2017 18:30:31 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492882230049400170' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'30215' | EOFError: EOF when reading a line
Sat, 22 Apr 2017 18:30:31 +0100 | ACCESS | session    | url:'http://localhost:60001/' id:'1492882230049400170' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'30215' | DISCONNECT
Sat, 22 Apr 2017 19:40:47 +0100 | INFO   | server     |  | Serving using application   : /usr/bin/python -u some-game.py
Sat, 22 Apr 2017 19:40:47 +0100 | INFO   | server     |  | Starting WebSocket server   : ws://adamk117-System-Product-Name:60001/
Sat, 22 Apr 2017 19:41:37 +0100 | ACCESS | session    | url:'http://localhost:60001/' id:'1492886497853153253' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' | CONNECT
Sat, 22 Apr 2017 19:41:39 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492886497853153253' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'31851' | Traceback (most recent call last):
Sat, 22 Apr 2017 19:41:39 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492886497853153253' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'31851' |   File "some-game.py", line 7, in <module>
Sat, 22 Apr 2017 19:41:39 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492886497853153253' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'31851' |     name = raw_input()
Sat, 22 Apr 2017 19:41:39 +0100 | ERROR  | stderr     | url:'http://localhost:60001/' id:'1492886497853153253' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'31851' | EOFError: EOF when reading a line
Sat, 22 Apr 2017 19:41:39 +0100 | ACCESS | session    | url:'http://localhost:60001/' id:'1492886497853153253' remote:'localhost' command:'/usr/bin/python' origin:'http://localhost:8000' pid:'31851' | DISCONNECT
Sat, 22 Apr 2017 21:24:55 +0100 | INFO   | server     |  | Serving using application   : /usr/bin/python -u some-game.py
Sat, 22 Apr 2017 21:24:55 +0100 | INFO   | server     |  | Starting WebSocket server   : ws://adamk117-System-Product-Name:60001/
Sat, 22 Apr 2017 21:25:44 +0100 | ACCESS | session    | url:'http://2.218.100.254:60001/' id:'1492892744330089973' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' | CONNECT
Sat, 22 Apr 2017 21:25:48 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892744330089973' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2245' | Traceback (most recent call last):
Sat, 22 Apr 2017 21:25:48 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892744330089973' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2245' |   File "some-game.py", line 7, in <module>
Sat, 22 Apr 2017 21:25:48 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892744330089973' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2245' |     name = raw_input()
Sat, 22 Apr 2017 21:25:48 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892744330089973' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2245' | EOFError: EOF when reading a line
Sat, 22 Apr 2017 21:25:48 +0100 | ACCESS | session    | url:'http://2.218.100.254:60001/' id:'1492892744330089973' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2245' | DISCONNECT
Sat, 22 Apr 2017 21:26:05 +0100 | ACCESS | session    | url:'http://2.218.100.254:60001/' id:'1492892765739192561' remote:'cpc102188-brmb9-2-0-cust74.1-3.cable.virginm.net.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' | CONNECT
Sat, 22 Apr 2017 21:26:16 +0100 | ACCESS | session    | url:'http://2.218.100.254:60001/' id:'1492892776065792178' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' | CONNECT
Sat, 22 Apr 2017 21:26:26 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892776065792178' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2259' | Traceback (most recent call last):
Sat, 22 Apr 2017 21:26:26 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892776065792178' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2259' |   File "some-game.py", line 7, in <module>
Sat, 22 Apr 2017 21:26:26 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892776065792178' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2259' |     name = raw_input()
Sat, 22 Apr 2017 21:26:26 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892776065792178' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2259' | EOFError: EOF when reading a line
Sat, 22 Apr 2017 21:26:26 +0100 | ACCESS | session    | url:'http://2.218.100.254:60001/' id:'1492892776065792178' remote:'SkyRouter.Home.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2259' | DISCONNECT
Sat, 22 Apr 2017 21:27:24 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892765739192561' remote:'cpc102188-brmb9-2-0-cust74.1-3.cable.virginm.net.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2258' | Traceback (most recent call last):
Sat, 22 Apr 2017 21:27:24 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892765739192561' remote:'cpc102188-brmb9-2-0-cust74.1-3.cable.virginm.net.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2258' |   File "some-game.py", line 7, in <module>
Sat, 22 Apr 2017 21:27:24 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892765739192561' remote:'cpc102188-brmb9-2-0-cust74.1-3.cable.virginm.net.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2258' |     name = raw_input()
Sat, 22 Apr 2017 21:27:24 +0100 | ERROR  | stderr     | url:'http://2.218.100.254:60001/' id:'1492892765739192561' remote:'cpc102188-brmb9-2-0-cust74.1-3.cable.virginm.net.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2258' | EOFError: EOF when reading a line
Sat, 22 Apr 2017 21:27:24 +0100 | ACCESS | session    | url:'http://2.218.100.254:60001/' id:'1492892765739192561' remote:'cpc102188-brmb9-2-0-cust74.1-3.cable.virginm.net.' command:'/usr/bin/python' origin:'http://2.218.100.254:8000' pid:'2258' | DISCONNECT
Fri, 28 Apr 2017 20:36:44 +0100 | INFO   | server     |  | Serving using application   : /usr/bin/python -u some-game.py
Fri, 28 Apr 2017 20:36:44 +0100 | INFO   | server     |  | Starting WebSocket server   : ws://adamk117-System-Product-Name:60001/
